app/globals.css

```
@@ -170,57 +210,63 @@
+    opacity: 0.035;
+    border-radius: inherit;
+    background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 200 200' xmlns='http://www.w3.org/2000/svg'%3E%3Cfilter id='noiseFilter'%3E%3CfeTurbulence type='fractalNoise' baseFrequency='0.65' numOctaves='3' stitchTiles='stitch'/%3E%3C/filter%3E%3Crect width='100%25' height='100%25' filter='url(%23noiseFilter)'/%3E%3C/svg%3E");
   }
 
   .panel {
-    @apply border border-border/70 bg-card/78 backdrop-blur-xl;
-    box-shadow: 0 22px 80px rgba(26, 36, 74, 0.06);
+    @apply border border-border/60 bg-card/75 backdrop-blur-xl;
+    box-shadow:
+      0 1px 2px rgba(30, 60, 40, 0.04),
+      0 16px 60px rgba(30, 60, 40, 0.05);
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)